<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileExport</class>
 <widget class="QDialog" name="FileExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>641</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="exportPage">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>601</width>
       <height>141</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>CSV出力ディレクトリ</string>
     </property>
     <widget class="QLabel" name="filenameLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>561</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>游ゴシック</family>
        <pointsize>11</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
  background-color:white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="exportButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>エクスポートする</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>キャンセル</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="exportOption">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>211</width>
       <height>151</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>出力データ選択</string>
     </property>
     <widget class="QRadioButton" name="cpuDataExportRadioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>游ゴシック</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>CPU検査データ</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="lanDataExportRadioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>游ゴシック</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>LAN検査データ</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>351</width>
       <height>151</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>データ出力範囲</string>
     </property>
     <widget class="QDateEdit" name="startDateEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>81</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>游ゴシック</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>18</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="time">
       <time>
        <hour>18</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>4022</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>9</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
     <widget class="QDateEdit" name="endDateEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>81</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>游ゴシック</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>9</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>1</month>
        <day>2</day>
       </datetime>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>4022</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>9</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>游ゴシック</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>開始日</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>游ゴシック</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>終了日</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="csvSelectButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>次へ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="csvCancelButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>キャンセル</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
